lms/djangoapps/courseware/views/views.py

```diff
@@ -888,9 +889,8 @@ def _progress(request, course_key, student_id):
         'masquerade': masquerade,
         'supports_preview_menu': True,
         'student': student,
-        'passed': is_course_passed(course, grade_summary),
```

Comment:
Previously, "passed" was used in the front end code to display the cert information. I moved this into the _get_cert_data method so that all logic about what message and buttons to display would be in the same place.